mappings/net/minecraft/client/render/VertexBuffer.mapping

```diff
@@ -15,40 +25,82 @@ CLASS none/bon net/minecraft/client/render/VertexBuffer
 	FIELD o building Z
 	METHOD <init> (I)V
 		ARG 0 initialSize
+	METHOD a getVertexState ()Lnone/bon$a;
 	METHOD a texture (DD)Lnone/bon;
 		ARG 0 u
 		ARG 1 v
+	METHOD a putPosition (DDD)V
+		ARG 0 x
+		ARG 1 y
+		ARG 2 z
 	METHOD a (FFF)V
 	METHOD a color (FFFF)Lnone/bon;
 		ARG 0 red
 		ARG 1 green
 		ARG 2 blue
 		ARG 3 alpha
+	METHOD a putColorMultipliers (FFFI)V
+		ARG 0 red
+		ARG 1 green
+		ARG 2 blue
+	METHOD a putColor (I)V
+		ARG 0 argb
 	METHOD a texture (II)Lnone/bon;
 		ARG 0 u
 		ARG 1 v
+	METHOD a putBrightness (IIII)V
+	METHOD a putColorRGBA (IIIII)V
```

Comment:
`putColor` ? Should be `putColorRgba` as per our standards otherwise.
The others could just be overloads.